src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -878,6 +879,13 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
         manager.register_pass<ov::intel_gpu::KVCacheFusion>();
         manager.register_pass<ov::intel_gpu::FullyConnectedConvertFusion>();
         manager.register_pass<ov::intel_gpu::TransposeFusion>(device_info.supports_immad);
+        bool enable_activations_scaling = false;
+        auto activations_scale_factor = config.get_property(ov::hint::activations_scale_factor);
+        if (!(activations_scale_factor == 0.f || activations_scale_factor == 1.f)) {
+            enable_activations_scaling = true;
+        }
+        if (enable_activations_scaling)
+            manager.register_pass<ov::intel_gpu::FullyConnectedPerLayerScaling>(activations_scale_factor);
```

Comment:
I think we can just return in the pass if the hint is not valid.

Reply:
updated